modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,32 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
 
 endif::nodes[]
 ifdef::virt[]
-Use the `LongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+Use the `LongLifecycle` or `DevKubeVirtRelieveAndMigrate` profile to enable the descheduler on a virtual machine. To ensure proper scheduling, create virtual machines with CPU and memory requests based on the expected load.
+
+[IMPORTANT]
+====
+Only one descheduler profile can be enabled at a time. Select the profile that best fits your workload needs.
```

Comment:
I suggest to mention that `SoftTopologyAndDuplicates` profile can be used in combination with `DevKubeVirtRelieveAndMigrate` to rebalance also to satisfy soft topology constraints when possible.
This can be useful in the HCP use case.